hackathon_site/dashboard/frontend/src/components/teamDetail/SimpleOrderTables/SimpleOrderTables.test.tsx

```diff
@@ -0,0 +1,132 @@
+import { fireEvent, render, within, act } from "testing/utils";
+import { mockPendingOrdersInTable, mockReturnedOrdersInTable } from "testing/mockData";
+import React from "react";
+import {
+    AdminReturnedItemsTable,
+    SimplePendingOrderFulfillmentTable,
+} from "./SimpleOrderTables";
+import { waitFor } from "@testing-library/react";
+
+describe("<SimplePendingOrderFulfillmentTable />", () => {
```

Comment:
These tests look good! But I think you should add one more test that tests whether the checkboxes are shown only when the order status is ```"Submitted"``` (and otherwise not shown)